RELEASING.md

```diff
@@ -0,0 +1,31 @@
+# Process for cutting release of this plugin
+
+As noted in this repository's README.md, the documentation and code at [https://github.com/openshift/jenkins-plugin](https://github.com/openshift/jenkins-plugin) always hosts the very latest version, including possibly pre-released versions that are still under test.
+The associated repository under the JenkinsCI project, [https://github.com/jenkinsci/openshift-pipeline-plugin](https://github.com/jenkinsci/openshift-pipeline-plugin), is only updated as part of cutting 
+official releases of this plugin.
+
+To cut a new release of this plugin, first perform a `git clone` of [https://github.com/jenkinsci/openshift-pipeline-plugin](https://github.com/jenkinsci/openshift-pipeline-plugin), 
+
+## Set up local repository to cut release:
+
+1. From the parent directory you've chosen for you local repository, run `git clone git@github.com:jenkinsci/openshift-pipeline-plugin.git`
+1. Change directories into `openshift-pipeline-plugin`, and run `git remote add upstream git://github.com/openshift/jenkins-plugin`
+1. Then pull the latest changes from [https://github.com/openshift/jenkins-plugin](https://github.com/openshift/jenkins-plugin) with the following:
+```
+git checkout master
+git fetch upstream
+git fetch upstream --tags
+git rebase upstream/master
+git push origin master
+git push origin --tags
+```
+
+## Submit the new release to the Jenkins organization
+
+1. Assumptions: your Git ID has push access to the two repositories for this plugin; your Jenkins ID (https://wiki.jenkins-ci.org/display/JENKINS/User+Account+on+Jenkins) is listed in https://github.com/jenkins-infra/repository-permissions-updater/blob/master/permissions/plugin-openshift-pipeline.yml
+1. Then run `mvn release:prepare release:perform`
+	- NOTE:  You'll minimally be prompted for the `release version`, `release tag`, and the `new development version` (typically a version indicator that ends with `-SNAPSHOT`).  Also, if we happen to need to depend on a SNAPSHOT version of openshift-restclient-java, you'll get prompted about moving off the SNAPSHOT version, but override this.  Stay on the SNAPSHOT version of openshift-restclient-java if recent updates to that dependency that we need have not yet been cut to a new FINAL release.
```

Comment:
OK I've attempted to clarify / clear up.  Let me know if it makes better sense to humans besides me :-)